### DISCUSSION TITLE
intercept lib

### DISCUSSION THREAD

**Discussion by Fizzbb** [CentaurusInfra/alnair#117] [Ideas]
line XXX, issues?

**Comment by CalvinXKY** [comment]
cuda_interpose里面dlsym函数：
1、看到omp threads有个issue，这个是什么问题？  
2、在该函数里面通过real_func[SYM_CU_INIT] = real_dlsym(handle, symbol)方式获取原函数，如果是非dlsym模式调用，这个函数即为NULL，是不是就会出现core dump？
